guidelines/REUSE.md

```diff
@@ -1,85 +1,100 @@
 # Reuse Philosophy
+
 We encourage reuse and patterns through references.
 
 ## What is reusable
+
 The following types are reusable, as defined by the spec:
 
-* Operations
 * Parameters
-* Responses
 * Models (or Schema Objects in general)
+* Responses
+* Operations (_Operations can only be remote references_)
 
 ## Reuse strategy
-When reusing components in an API design, a pointer is created from the definition to target design.  The references are maintained in the structure, and can be updated by modifying the source definitions.  This is different from a "copy on design" approach where references are injected into the design itself.
 
-The reuse technique is transparent between JSON or YAML and is lossless when converting between the two.
+When reusing components in an API design, a reference is created from the definition to target entity. The references are maintained in the structure, and can be updated by modifying the source definitions. This is different from a "copy on design" approach where references are injected into the design itself.
```

Comment:
@whitlockjc I don't get this sentence:
> The major difference between JSON References and OpenAPI references is that JSON References can point to anything, anywhere but OpenAPI references are intended only to point to definitions as defined in the section above.

With remote URL references, you could potentially point anywhere. For that matter, with remote relative references, you can point to files anywhere in the file system, in theory. The convention aspect of local references is already covered in #local-references.